semantic_conventions/trace/cloudevents.yaml

```diff
@@ -0,0 +1,40 @@
+groups:
+  - id: cloudevents
+    prefix: cloudevents
```

Comment:
```suggestion
  - id: cloudevent
    prefix: cloudevent
```

We are describing just a single event here.

Reply:
The "brand" name is `cloudevents` so I think we should stick to it, no?